### PR TITLE
feat: New spec "ls -lZ /var/lib/rsyslog" and the parser

### PR DESCRIPTION
**Pull request by huali027**
feat: New spec "ls -lZ /var/lib/rsyslog" and the parser 

Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce Please help to review, thanks.

**Review by xiangce — APPROVED** [review]
Good to me.

**Comment by xiangce** [comment]
@ryan-blakley - Could you please have a check as well?

File: insights/parsers/ls_var_lib_rsyslog.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""
+LsVarLibRsyslog - command ``ls -lZ  /var/lib/rsyslog``
+======================================================
+"""
+
+
+from insights.specs import Specs
+from insights import CommandParser, parser, FileListing
```
Comment (ryan-blakley) [inline insights/parsers/ls_var_lib_rsyslog.py:8]: Could you please change these to full path imports like below?

```
from insights.core import CommandParser, FileListing
from insights.core.plugins import parser
from insights.specs import Specs
```

**Review by ryan-blakley — CHANGES_REQUESTED** [review]

File: insights/parsers/ls_var_lib_rsyslog.py (hunk 1 above)
Comment (huali027) [reply insights/parsers/ls_var_lib_rsyslog.py]: > from insights.core import CommandParser, FileListing
> from insights.core.plugins import parser
> from insights.specs import Specs

@ryan-blakley updated, please help to check, thanks.

Comment (ryan-blakley) [reply insights/parsers/ls_var_lib_rsyslog.py]: Thank you, looks good.

**Review by ryan-blakley — APPROVED** [review]

**Pull request merged**